node_modules/oae-ui/lib/api.js

```diff
@@ -703,7 +703,37 @@ var getTenantSkinVariables = module.exports.getTenantSkinVariables = function(te
     var tenantVariables = _getTenantSkinVariableValues(tenantAlias);
 
     // Merge the default and tenant variables, tenant variables having precedence
-    return _.extend({}, defaultVariables, tenantVariables);
+    var resolvedVariables = _.extend({}, defaultVariables, tenantVariables);
+
+    // Template the institutional logo url
+    _renderDynamicValue(resolvedVariables, 'institutional-logo-url', {'tenantAlias': tenantAlias});
+
+    return resolvedVariables;
+};
+
+/**
+ * Render the key of the specified object through underscore.js template, with
+ * the given data. If the key does not exist on the object, then this function
+ * will do nothing
+ *
+ * @param  {Object}     obj     The arbitrary object whose value to render
+ * @param  {String}     key     The key of the specified object to replaced with a rendered version
+ * @param  {Object}     data    The data for the template
+ * @api private
+ */
+var _renderDynamicValue = function(obj, key, data) {
+    var value = obj[key];
+    if (value) {
+        try {
+            obj[key] = _.template(value)(data);
```

Comment:
In the past we haven't exposed underscore templates to tenant admins as they could enter stuff like:

```javascript
var skinConfig = {
    'institutional-logo-url': '   '<% while (true) { console.log('LOL'); } %>'
};
```

and effectively block the server